packages/website/app/styles/root.css

```diff
@@ -0,0 +1,47 @@
+:root {
+    --gray: #d2d0ce;
+    --gray-light: #edebe9;
+}
+
+li {
+    list-style: none;
+    font-size: 18px;
+}
+
+a {
+    border-radius: 4px;
+    text-decoration: none;
+    height: 30px;
+    padding: 3px 5px;
+    transition: all 0.3s;
+}
+
+a:hover {
+    text-decoration: underline;
+}

```

Comment:
Same, unocss is possible. And they are mostly local to `headerNav` and `articleBox`. Moreover `text-decoration: none` is removed by default.